test/cmd/newapp.sh

```diff
@@ -82,6 +82,20 @@ os::cmd::expect_success_and_not_text 'oc new-app ruby-helloworld-sample --param
 os::cmd::expect_success_and_text 'oc new-app php PASS=one,two=three -o yaml' 'value: one,two=three'
 os::cmd::expect_success_and_not_text 'oc new-app php PASS=one,two=three -o yaml' 'no longer accepts comma-separated list'
 
+# check that we can populate template parameters from file
+param_file="${OS_ROOT}/test/testdata/test-cmd-newapp-params.env"
+os::cmd::expect_success_and_text "oc new-app ruby-helloworld-sample --param-file ${param_file} -o jsonpath='{.items[?(@.kind==\"DeploymentConfig\")].spec.template.spec.containers[0].env[?(@.name==\"MYSQL_PASSWORD\")].value}'" 'thisisapassword'
```

Comment:
> How can I cause access denied if the tests are run as root? 

Ok, I'm ok with skipping this one.  :)

> Should empty file really cause an error?

No, it's ok to just interpret that as "no env vars to set", but still should have a test.